### PR TITLE
feat(woo): replace orderbook with orderbookupdate 

### PR DESCRIPTION
**Pull request by sc0Vu**
The frequency to push data of orderbook topic is 1s. However, it's 200ms to orderbookupdate. In this PR, I replace orderbook with orderbookupdate in `watchOrderBook`.

### CODE REVIEW

File: ts/src/pro/woo.ts [hunk 1]
```diff
@@ -90,33 +90,45 @@ export default class woo extends wooRest {
...
+         * @see https://docs.woo.org/#orderbookupdate
          * @description watches information on open orders with bid (buy) and ask (sell) prices, volumes and other data
          * @param {string} symbol unified symbol of the market to fetch the order book for
          * @param {int} [limit] the maximum amount of order book entries to return.
          * @param {object} [params] extra parameters specific to the exchange API endpoint
          * @returns {object} A dictionary of [order book structures]{@link https://docs.ccxt.com/#/?id=order-book-structure} indexed by market symbols
          */
         await this.loadMarkets ();
-        const name = 'orderbook';
+        const name = 'orderbookupdate';
```
Comment (carlosmiei) [inline ts/src/pro/woo.ts:101]: @sc0Vu can't we add an option to let the users choose if needed?

Comment (sc0Vu) [reply ts/src/pro/woo.ts:101]: @carlosmiei We can support both ways.

File: ts/src/pro/woo.ts [hunk 2]
```diff
@@ -90,33 +90,49 @@ export default class woo extends wooRest {
...
          * @param {string} symbol unified symbol of the market to fetch the order book for
          * @param {int} [limit] the maximum amount of order book entries to return.
          * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {string} [params.method] either (default) 'orderbook' or 'orderbookupdate'
          * @returns {object} A dictionary of [order book structures]{@link https://docs.ccxt.com/#/?id=order-book-structure} indexed by market symbols
          */
         await this.loadMarkets ();
-        const name = 'orderbook';
+        const defaultMethod = 'orderbook';
+        const method = this.safeString (params, 'method', defaultMethod);
```
Comment (carlosmiei) [inline ts/src/pro/woo.ts:103]: @sc0Vu next time, please use `handleOptionAndParams` to read this

**Comment by carlosmiei** [comment]
DEMO

```
Python v3.12.3
CCXT v4.3.70
woo.watchOrderBook(BTC/USDT,None,{'method': 'orderbookupdate'})
{'asks': [[64543.2, 0.085198], [64546.08, 0.077524], [64550.96, 0.038094], [64552.27, 0.006196], [64552.64, 0.06972], [64554.1, 0.211841], [64554.8, 0.155028], [64555.1, 0.522027], [64555.9, 0.542537], [64556.0, 0.1461], [64556.45, 0.092961], [64556.8, 0.0001], [64556.9, 0.0001], [64557.1, 0.01], [64558.0, 0.146193], [64559.0, 0.001], [64559.1, 0.02327], [64559.2, 0.092305], [64559.8, 0.15494], [64560.0, 0.1461], [64560.2, 0.012477], [64560.33, 0.012393], [64560.4, 1.5e-05], [64560.9, 0.00191], [64561.0, 0.0031], [64561.03, 0.139441], [64561.2, 2.8e-05], [64562.0, 0.1461], [64562.6, 0.038727], [64562.7, 0.132047], [64562.8, 0.038726], [64563.1, 0.15493], [64563.56, 0.018589], [64563.6, 0.023329], [64564.2, 0.038727], [64564.3, 0.250837], [64564.4, 0.474988], [64564.5, 0.295], [64564.6, 0.186159], [64565.2, 0.155003], [64565.5, 0.02], [64565.82, 0.015], [64565.9, 0.220853], [64566.1, 0.020024], [64566.2, 0.154946], [64566.8, 0.061984], [64566.9, 0.04259], [64567.5, 0.040692], [64568.1, 0.020024], [64570.5, 0.05605], [64570.9, 0.16597], [64571.5, 0.154988], [64572.0, 0.103348], [64572.1, 3.181293], [64572.8, 0.30987], [64573.3, 0.086126], [64573.7, 0.680605], [64574.1, 0.020024], [64574.7, 0.15873], [64574.8, 1.134317], [64575.5, 0.022299], [64575.6, 0.15516], [64576.1, 0.163052], [64576.4, 0.02], [64577.3, 0.07748], [64577.8, 0.020024], [64578.0, 0.077454], [64578.3, 0.54238], [64578.6, 0.022989], [64578.7, 0.022298], [64579.2, 0.032354], [64579.9, 0.158745], [64580.1, 0.351024], [64580.5, 0.29072], [64581.1, 0.061731], [64581.6, 0.468934], [64581.8, 0.474988], [64584.1, 0.020024], [64584.5, 0.023244], [64584.6, 0.022747], [64585.3, 0.020024], [64585.6, 0.890221], [64585.8, 0.023243], [64586.1, 0.020024], [64586.6, 0.29503], [64586.69, 0.309903], [64587.3, 0.15873], [64588.1, 0.020024], [64590.7, 0.026697], [64591.9, 0.056024], [64592.9, 0.177015], [64593.0, 0.063545], [64593.2, 0.295], [64593.5, 0.071963], [64593.9, 0.15873], [64595.3, 0.965251], [64595.4, 1.608751], [64595.7, 0.027663], [64597.6, 0.359471], [64598.2, 0.103], [64599.0, 0.026595], [64599.3, 0.474988], [64599.6, 0.158745], [64599.9, 0.849886], [64600.0, 1.248291], [64600.2, 0.059263], [64600.6, 0.035284], [64601.2, 1.450785], [64602.0, 0.034925], [64603.9, 0.177352], [64604
```

File: ts/src/pro/woo.ts (hunk 2 above)
Comment (sc0Vu) [reply ts/src/pro/woo.ts]: No problem!

**Pull request merged**